### DISCUSSION TITLE
Neue Würfelmethode in der Topleiste bei Mare Nostrum

### DISCUSSION THREAD

**Discussion by runeb68** [Gorthian/HeXXen1733_Custom_Character_Sheet#39] [Praise & Blame]
Moin

Die neue Methode sieht zwar sehr schön aus, ich empfinde sie aber als umständlicher.

1. Man muß ein- bis zweimal mehr klicken. Einmal auf die Fertigkeit oder zweimal mit der Auswahlliste + Wurf.

2. Wenn es keine eigenen Knöpfe für die Würfe mehr gibt kann man diese auch nicht in die Schnell-Makro-Leiste ziehen. Diese Möglichkeit hat Roll20 meines Wissens erst dieses Jahr eingeführt. Man nehme einen beliebigen Würfelknopf auf einem Charakterbogen und ziehe ihn nach unten. Dann erscheint ein grauer Bereich am unteren Rand des Bogens wo auch bereits vorhandene Knöpfe zu sehen sind. Es erscheint ein Knopf "Charaktername:Technischerwurfname".  Wenn man dann die "Macro Quick Bar" einschaltet erscheint auf der Karte unter den Avataren eine Leiste wo man die Buttons umbenennen, umsortieren und einfärben und natürlich zum Würfeln anklicken kann.

Gruß, Rune.

Ich hatte es bei den Vorschlägen und Fehlermeldungen  (Issues) schon geschrieben:
   Mir - Und meiner Gruppe - gefällt der Bogen sehr gut... 
       Deshalb bemühe ich mich auch so darum dazu bei zu tragen.

Screenshot mit Quickmakroleiste:

![image](https://user-images.githubusercontent.com/78030873/115487425-f2d81000-a258-11eb-89e9-02adb456906c.png)

P.S. in dem Bild sieht man auch das ich bei den Angriffen in den Wurftext noch Waffenwerte eingefügt habe...

**Comment by Gorthian** [comment]
Die neue Methode ist auch umständlicher und beide von Dir genannte Punkte sind korrekt. 
Leider sehe ich aktuell aber dennoch keine andere Option, als die Auslagerung der Würfe in die Top-Leiste.

Das Grundproblem ist die komplexer werdende Berechnung des Januspools. Mit Mare Monstrum kamen weitere Kräfte, Gebrechen, Whatever dazu, die speziellen Proben dauerhaft einen Bonus oder Malus geben können.
Losgelöst von den Malusstufen, kann es also z.B. passieren, dass alle meine Proben auf Redekunst ein +1 bekomen, während alle Proben die auf Geschicklichkeit basieren -1 bekommen.

Ich muss also den Januspool basierend auf der ausgwähöten Fertigkeit vor jedem Wurf neu berechnen.
Und das kann Roll20 aktuell meines Wissens nach nicht. 
Ein Button kann entweder eine Berechnung ausführen, oder einen Wurf auslösen. aber nicht beides.
Und aus einer Berechnung heraus, kann ich wiederum keinen Wurf initiieren. (Oder umgekehrt)

Daher kam leider diese "Krücke" über die Top-Bar ins Spiel: Ein Klick auf eine Fertigkeit führt jetzt die Neuberechnung des Januspools durch, und belegt die entsprechenden Felder für die eigentliche Probe vor. Diese muss dann eben über einen zweiten Klick ausgeführt werden.

100% zufrieden bin ich damit auch nicht, aber für mich überwiegt der Nutzen der automatischen Berechnung des Januspools gegenüber dem zusätzlichen Klick beim Würfeln.
